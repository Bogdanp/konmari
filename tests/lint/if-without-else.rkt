#lang racket/base

(if #t
    (foo))
