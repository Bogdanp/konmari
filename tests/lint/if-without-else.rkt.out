/if-without-else.rkt:3:1:error:if expressions must contain one expression for the then-branch and another for the else-branch
